Source/WebCore/platform/mediarecorder/cocoa/AudioSampleBufferCompressor.mm

```diff
@@ -141,6 +144,8 @@
                 return GenericPromise::createAndReject();
             }
         }
+        m_lastError = noErr;
```

Comment:
m_lastError is probably only used in `m_serialDispatchQueue`, can we add a threading annotation?